src/implementation/Client/HTTPClient/HTTPClient.ts

```diff
@@ -107,12 +109,21 @@ export default class HTTPClient implements IClient {
     return this.isInitialized;
   }
 
+  async _startAwaitSidecarStarted(): Promise<void> {
+    await DaprClient.awaitSidecarStarted(async () => await HTTPClientSidecar.isStarted(this));
+  }
+
   async stop(): Promise<void> {
     this.httpAgent.destroy();
     this.httpsAgent.destroy();
   }
 
   async start(): Promise<void> {
+    await this._startAwaitSidecarStarted();
```

Comment:
Do we need to move it to a function?

Reply:
What do you mean?

Reply:
Since the function implementation is trivial and not being used by anyone else, shall we not have a function and put the contents of `startAwaitSidecarStarted` in the `start` function directly?